libraries/bxdf/standard_surface.mtlx

```diff
@@ -63,21 +63,21 @@
             doc="The roughness of the clear-coat reflections. The lower the value, the sharper the reflection." />
       <input name="coat_anisotropy" type="float" value="0.0" uimin="0.0" uimax="1.0" uiname="Coat Anisotropy" uifolder="Coat" uiadvanced="true"
             doc="The amount of directional bias, or anisotropy, of the clear-coat layer." />
-      <input name="coat_rotation" type="float" value="0.0" uimin="0.0" uimax="1.0" uiname="Coat Rotation" uifolder="Coat" uiadvanced="true"
+      <input name="coat_rotation" type="float" value="0.0" uisoftmin="0.0" uisoftmax="1.0" uiname="Coat Rotation" uifolder="Coat" uiadvanced="true"
             doc="The rotation of the anisotropic effect of the clear-coat layer." />
-      <input name="coat_IOR" type="float" value="1.5" uimin="0.0" uimax="3.0" uiname="Coat Index of Refraction" uifolder="Coat"
+      <input name="coat_IOR" type="float" value="1.5" uimin="0.0" uisoftmax="3.0" uiname="Coat Index of Refraction" uifolder="Coat"
             doc="The index of refraction of the clear-coat layer." />
       <input name="coat_normal" type="vector3" defaultgeomprop="Nworld" uimin="0,0,0" uimax="1,1,1" uiname="Coat normal" uifolder="Coat"
```

Comment:
Old error: normal should probably not have any min/max, as it's never given by the user. And the range given here is incorrect. So I think this could be removed. Maybe you could fix this as well in this PR?